academy/3-ibc/6-ica.md

```diff
@@ -16,635 +15,297 @@ tags:
 
-    ...
+Several relevant definitions relating to ICA are as follows:
 
-    // if there is an active channel for this portID / connectionID return an error
-    activeChannelID, found := k.GetOpenActiveChannel(ctx, connectionID, portID)
-    if found {
-        return sdkerrors.Wrapf(icatypes.ErrActiveChannelAlreadySet, "existing active channel %s for portID %s on connection %s for owner %s", activeChannelID, portID, connectionID, owner)
-    }
+**Host Chain:** the chain where the Interchain account is registered. The host chain listens for IBC packets from a controller chain which should contain instructions (e.g. cosmos SDK messages) which the Interchain account will execute.
 
-    ...
+**Controller Chain:** the chain that registers and controls an account on a host chain. The controller chain sends IBC packets to the host chain to control the account.
 
-    connectionEnd, err := k.channelKeeper.GetConnection(ctx, connectionID)
+<HighlightBox type="info">
 
-    ...
+The Interchain accounts application module is structured to **support the ability to exclusively enable controller or host functionality**. This can be achieved by simply omitting either the controller or host `Keeper` from the Interchain account's `NewAppModule` constructor function, and mounting only the desired submodule via the `IBCRouter`. Alternatively, [submodules can be enabled and disabled dynamically using on-chain parameters](https://ibc.cosmos.network/main/apps/interchain-accounts/parameters.html).
 
-    msg := channeltypes.NewMsgChannelOpenInit(portID, string(versionBytes), channeltypes.ORDERED, []string{connectionID}, icatypes.PortID, icatypes.ModuleName)
+</HighlightBox>
 
-    ...
-}
-```
+**Interchain account (ICA):** an account on a host chain. An Interchain account has all the capabilities of a normal account. However, rather than signing transactions with a private key, a controller chain's authentication module will send IBC packets to the host chain which signal what transactions the Interchain account should execute.
 
-The `portID` will be the address of the **interchain account Owner** prefixed by the default port prefix of the interchain accounts controller submodule `icacontroller-`. The ICA module assumes that there is already an IBC connection established between the host and controller chains. As part of `RegisterInterchainAccount`, it will open an `ORDERED` channel.
+**Interchain account owner:** an account on the controller chain. Every Interchain account on a host chain has a respective owner account on a controller chain. This could be a module account or other analogous account, not just a regular user account.
```

Comment:
```suggestion
**Interchain account owner:** an account on the controller chain. Every Interchain account on a host chain has a respective owner account on a controller chain. This is not a regular user account, but more similar to a module account or other analogous account.
```

Reply:
@charleenfei 

> This is not a regular user account...

Does "This" refer to the account on the host chain?

Reply:
No it refers to an account on the controller chain. You're right @charleenfei it reads a bit clunky but the ICA owner account could still be regular user account, so what about:

"This could be a module account (in Cosmos SDK chains )or analogous accounts, it is not strictly limited to regular user accounts."

Reply:
that sgtm :)

Reply:
```suggestion
**Interchain account owner:** an account on the controller chain. Every Interchain account on a host chain has a respective owner account on a controller chain. This owner account could be a module account (in Cosmos SDK chains ) or analogous accounts, it is not strictly limited to regular user accounts.
```